src/install/default-trusted-dependencies.txt

```diff
@@ -73,6 +73,10 @@
 @pulumi/awsx
 @pulumi/command
 @pulumi/kubernetes
+@pulumi/postgresql
```

Comment:
why was @pulumi/postgresql and these others added? i checked postgresql and it does not have an install script